Лабораторная работа 2/task2.py

```diff
@@ -0,0 +1,16 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+
+money_capital = 0
+sum_salary = salary * months
+sum_spend = 0
+
+for i in range(months+1):
```

Comment:
Хорошая работа!
Но нужно немного исправить цикл.
Ошибка заключается в том, что в цикле `for` переменная `i` увеличивается от 0 до 10 включительно, то есть всего выполняется 11 итераций вместо необходимых 10. Это приводит к тому, что сумма трат рассчитывается неправильно. Чтобы исправить ошибку, нужно изменить условие цикла:
```suggestion
for i in range(months):
```
Теперь цикл будет выполняться ровно 10 раз, как требуется.

PS.
Когда вы используете функцию `range()`, она генерирует последовательность чисел, начиная с указанного начального значения (по умолчанию 0) и заканчивая значением, указанным в аргументе, но не включая его. Например:
```
range(10) - создаст последовательность чисел от 0 до 9 (включительно).
```
```
range(1, 4) создаст последовательность 1, 2, 3.
```
Таким образом, когда вы хотите выполнить цикл определённое количество раз, вам нужно указать это число в качестве верхнего предела функции `range()` (без включения этого числа). В вашем случае, если нужно выполнить цикл 10 раз, следует использовать:
```
range(months)
```